### DISCUSSION TITLE
comparing to...

### DISCUSSION THREAD

**Discussion by eleaner** [moosefs/moosefs#483] [General]
Hi everyone

For my home lab, I run moose on a single machine with multiple hard drives as easily expandable storage that is bitrot protected.
It runs rather awesome, even on elderly hardware.

Last week I tried to compare it to seaweeds running on the same machine (considering built-in ec it sounded like a good idea)
but moose was easily twice as fast in reads and writes (and it was already loaded with data)
Why is that?

Has anyone also tried some home-lab comparison?

**Comment by onlyjob** [comment]
See also #370.

**Comment by eleaner** [comment]
good point; I tested on rather large files

**Comment by eleaner** [comment]
@onlyjob 
I did a very crude test
I sat s3ql (older version) on both moose and seaweed
s3ql stores data in 10MB chunks by default

s3ql on moose sits directly on the filesystem
s3ql on weed sits on weeds own s3 (I am not sure if it could be a benefit or disadvantage, but I assumed S3ql was built for s3)

tested with dd on mounted s3ql, the differences in the performance are negligible with slight favour in the direction of weed
considering EC implementation, that's a massive upside, but I believe I would need 15+HDD to benefit from that (10+4 EC implementation)

on the negative weed seems to be less easy to understand what is going on in case something is wrong
